### ISSUE TITLE
GET Annotation implementation

### ISSUE THREAD

**Issue by RicardoMonteiroSimoes** [hephaistos-io/Fabulinus#2]
One idea of implementation would be this one:

```Java
@GET
private String name;

public String getName(){
    return name;
}
```

The `@GET` annotation would automatically assign itself to the corresponding `getXYZ` function for the variable. Edge cases of a `@GET` would need to be defined - should an Exception be thrown or just default to using reflection to access the value?

_Originally posted by @RicardoMonteiroSimoes in https://github.com/hephaistos-io/Fabulinus/issues/1#issuecomment-572205632_

**Comment by RicardoMonteiroSimoes** [comment]
Create an Annotation called `@GET` that allows the following:
### Case 1

Here it will handle getting the value of the variable itself trough reflection

```Java
@GET
private String name;
```

### Case 2

Here it will "autodetect" the function called `get<variableName>` that returns the correct type and use that one to get the value.

```Java
@GET
private String name;

....

public String getName();
```

### Case 3

Here it will search for the function called "getCapitalizedName" and use that to retrieve the value.

```Java
@GET(function = "getCapitalizedName")
private String name;

....

public String getCapitalizedName()
```


### Exceptions

Case 1,2 and 3 shall throw a RuntimeException if the variable is final

Case 2 and 3 shall throw a RuntimeException if the required function is not present

**Comment by RicardoMonteiroSimoes** [comment]
Closed as off #3
